src/index.html

```diff
@@ -11,7 +11,199 @@
+          <input
+            type="text"
+            name="name"
+            id="name"
+            required
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            name="age"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="date">Full date of birth:</label>
+          <input
+            type="date"
+            name="date"
+            id="date"
+          >
+        </div>
+
+        <div class="form-fieldLast">
+          <label for="agreement">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            id="email"
+            required
+          >
+        </div>
+
+        <div class="form-fieldLast">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            required
+            minlength="6"
+            maxlength="32"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+
+
+          <input
+            type="radio"
+            name="cats"
+            id="catsy"
+          >
+
+          <label for="catsy">Yes</label>
+
+          <input
+          type="radio"
+          name="cats"
+          id="catsn"
+          >
+
+          <label for="catsn">No</label>
+
+        </div>
+        <div class="form-field">
+          <label for="color">What is your favorite color?</label>
+
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="datatime">What time do you go to bed?</label>
+
+          <input
+            type="time"
+            name="datatime"
+            id="datatime"
+            step="1"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="brand">What are your favourite brand of cars?</label>
+
+          <select
+            name="brand"
+            id="brand"
+            multiple
+          >
+            <option value="BMV">BMV</option>
+
+            <option value="Audi">Audi</option>
+
+            <option value="Lada">Lada</option>
+
+            <option value=" "> </option>
```

Comment:
No need for a single line elements
```suggestion
            <option value="BMV">BMV</option>
            <option value="Audi">Audi</option>
            <option value="Lada">Lada</option>
            <option value=" "> </option>
```